doc/manipulating_visualizing/data_preparation.rst

```diff
@@ -98,11 +109,11 @@ Sometimes, some custom preprocessing of data is necessary. For instance
 we can restrict a dataset to the first 100 frames. Below, we load
 a resting-state dataset with :func:`fetch_fetch_nyu_rest()
 <nilearn.datasets.fetch_nyu_rest>`, restrict it to 100 frames and
-build a brand new Nifti-like object to give it to the masker. Though it
-is possible, there is no need to save your data in a file to pass it to a
+build a brand new Nifti-like object to give it to the masker. Although
+possible, there is no need to save your data to a file to pass it to a
 :class:`NiftiMasker`. Simply use `nibabel
 <http://nipy.sourceforge.net/nibabel/>`_ to create a :ref:`Niimg <niimg>`
-in memory:
+dynamically in memory:
```

Comment:
I am not convinced dynamically adds anything here, especially given our target audience.